src/main/java/com/imaee/propinq/neighborhood/controllers/interfaces/INeighborhoodController.java

```diff
@@ -0,0 +1,35 @@
+package com.imaee.propinq.neighborhood.controllers.interfaces;
+
+import com.imaee.propinq.neighborhood.controllers.requests.NeighborhoodRequest;
+import com.imaee.propinq.neighborhood.controllers.responses.NeighborhoodResponse;
+import jakarta.validation.Valid;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RequestMapping("/neighborhood")
```

Comment:
Por convención, los endpoints deben ser plurales jijiji :V

```suggestion
@RequestMapping("/neighborhoods")
```